src/app/account/preferences.tsx

```diff
@@ -86,6 +88,8 @@ const AccountPreferences = () => {
     label: i.instrument.name,
   }));
 
+  const isTrivselOmbud = corps?.roles.some(role => role.name === "Trivselombud");
```

Comment:
Try to keep the code in English. Although in this case it might be acceptable as we are looking for a role called "Trivselombud".